src/peft/tuners/lora/model.py

```diff
@@ -667,48 +668,72 @@ def unload(self):
         """
         return self._unload_and_optionally_merge(merge=False)
 
-    def _adapter_names_to_ids(self):
-        adapter_names = ["base"] + list(self.peft_config.keys())
-        return {adapter_name: index for index, adapter_name in enumerate(adapter_names)}
+    # def _adapter_names_to_ids(self):
+    #     adapter_names = ["base"] + list(self.peft_config.keys())
+    #     return {adapter_name: index for index, adapter_name in enumerate(adapter_names)}
+
+    # def get_adapter_indices(self, adapter_names, return_tensor=True):
```

Comment:
this util is required by end user so that they can get the adapter indices by calling `model.get_adapter_indices(adapter_names)`. an example is shown below:

```
def get_model_pred(prompts, adapter_names):
    model.eval()
    adapter_indices = model.get_adapter_indices(adapter_names).cuda()
    inputs = tokenizer(prompts, return_tensors="pt", padding=True)
    if "token_type_ids" in inputs:
        del inputs["token_type_ids"]
    inputs = {k:v.cuda() for k,v in inputs.items()}
    outputs = model.generate(**inputs,
                             adapter_indices=adapter_indices,
                                 max_new_tokens=128,
                                 temperature=0.2,
                                 top_k=50,
                                 top_p=0.95,
                                 do_sample=True,
                                 repetition_penalty=1.1,
                                 eos_token_id = tokenizer.eos_token_id)
    return tokenizer.batch_decode(outputs, skip_special_tokens=False)
```